bootstrap/baremetal.lokocfg

```diff
@@ -25,6 +25,9 @@ cluster "bare-metal" {
   install_pre_reboot_cmds = var.install_pre_reboot_cmds
   node_specific_labels = var.node_specific_labels
   wipe_additional_disks = true
+  # Change this to 'true' to prevent the worker from being reprovisioned due to configuration changes.
+  # Using 'true' means the worker keeps its current state (e.g., storage volumes) but things like new SSH keys need to be added manually.
+  ignore_worker_changes = false
```

Comment:
We don't need to mention that change it to `true`  as that is the default behavior. In fact `ignore_worker_changes` shouldn't be present if user wants to set it to true.

We could mention remove this variable if user wants the workers to keep current state.

Reply:
Removing the variable is a bit opaque since the variable's existence and default behavior may be unknown when one looks at the file.